### DISCUSSION TITLE
Additional language support

### DISCUSSION THREAD

**Discussion by Shaarigan** [LinSoftWin/Photoshop-CC2022-Linux#98] [Ideas]
Hey,

I somehow  successfully installed CC2021 from your script,  except for the .desktop file that wasn't created for some reason. But PS started properly. There is only one thing that could be helpful, when there would be other languages available. Maybe as an optional download?

I know how to add/replace the default one but I don't have the appropriate files. At least the major European ones (German, Spanish, French) would be nice. 

Thank you!

**Comment by MiMillieuh** [comment]
This is not planned for now but eventually

**Comment by DomiLuebben** [comment]
@MiMillieuh @Shaarigan here are the German language files. Just put them in the "locales" folder and restart Photoshop. Then change language in preferences -> interface.
@MiMillieuh just include the language files in the archive that's downloaded by the script, so the Photoshop installation on Linux can be used in English and German, depending on the system. These files are from Photoshop CC 2023, but work perfectly fine with 2021 and 2022.

https://drive.google.com/file/d/1FcWp6z02ccAPyESJrerIy7UPJN1aYwVR/view?usp=share_link

**Comment by MiMillieuh** [comment]
[Locales.zip](https://github.com/MiMillieuh/Photoshop-CC2022-Linux/files/10852698/Locales.zip)

Here are all the language for now. I might include them later

Edit : I'll add all language to the custom PS version so the user can change it in the settings.

**Reply by DomiLuebben** [reply]
Wow, that's a lot of languages! Great, I think you didn't even need my help for that ;)

**Reply by Shaarigan** [reply]
Thank you, I really appreciate it!
